src/Analyzers/CSharp/Analyzers/UseCollectionExpression/CSharpUseCollectionExpressionForFluentDiagnosticAnalyzer.cs

```diff
@@ -313,11 +313,12 @@ void AddFinalMatch(ExpressionSyntax expression)
                             memberAccess.Expression.WithoutTrailingTrivia(),
                             memberAccess.OperatorToken.WithoutTrivia(),
                             memberAccess.Name.WithoutLeadingTrivia()))),
-                    UseSpread: true));
+                    UseSpread: true,
+                    UseKeyValue: false));
```

Comment:
added a new required parameter saying if we were creating a `k:v` parameter in teh final collection.  most places pass false.  some places pass through a computed value from an inner location.  and some places explicit pass true if we want that.

a few places that pass false will be updated to 'true' in followup PRs where i add support for more cases (like fluent/builders).